src/error.c

```diff
@@ -147,10 +147,10 @@ static Obj FuncUpEnv(Obj self, Obj args)
         depth = INT_INTOBJ(ELM_PLIST(args, 1));
     }
     else {
-        ErrorQuit("usage: UpEnv( [ <depth> ] )", 0L, 0L);
```

Comment:
I'm guessing changes like this (the change of `( [` to `([` are some kind of scripting error?

Reply:
Ouch, indeed: I think my regex was bad (I wrote something like `(Error.*)\( *([^;]+), *0L *, *0L* \);` for the capturing -- clearly the first group should have been something like `(Error[A-Za-z]*)`... *sigh*. I'll fix this. Thank you for looking carefully.

Reply:
Seems in addition to UpEnv and DownEnv, also QUIT and FORCE_QUITE were affected, but that's it (determined by grepping the diff between this PR and master). Will push a fix.